examples/hello/TestFSharp/TestFSharp.fsx

```diff
@@ -0,0 +1,10 @@
+﻿#r "../../../bin/ulibfs.dll"
+#r "bin/Debug/TestFSharp.dll"
+
+open TestFSharp
+
+let a = pass () () (fun x y -> x = y)
+let b = pass 5 2 (fun x y -> x = y)
+
+// TODO: 
+//let b = fail () () () () (fun x y -> x = y)
```

Comment:
And how do these differ from .repl.fsx?

Reply:
No difference. I've renamed those files to plain "fsx". See my previous comment.